styles/design-settings/carnival/parts/_table-settings.scss

```diff
@@ -200,6 +200,15 @@
     )
 ));
 
+@include add_settings((
+  AlternateShading: (
+        _selectors: ('.alternate-shading'),
+        LightGrayShading: (
+          background-color: (_ref:'colorMap:::tableRowGreyShadingColor'),
```

Comment:
Shading color might be different for other books, so it will be better to add settings directly in book.scss file.